v0.17.1.6-freebsd/hyc/monero-freebsd-0.17-build.assert

```diff
@@ -0,0 +1,409 @@
+--- !!omap
+- out_manifest: '1ad59103c9ea3a2256c8f4b7066cefd7ce2ebd2b52360edb539115c80a7e6ee1  monero-x86_64-unknown-freebsd-v0.17.1.6.tar.bz2
+
+    '
```

Comment:
Something is weird with the indentation here. Not really important, but it throws off the verification script.

Reply:
I've been getting this on the last few releases.  I have just been manually removing them.

Reply:
Strange, since these are produced by the gitian build scripts, and they haven't changed recently. ??

Reply:
I checked back to v0.16, it's been the same in every release. It seems to screw up whenever there's only a single output file. Since we build both 32 and 64bit for everything else, (windows, osx, android) this only happens with freebsd since we only build a 64bit there. So this is a bug in the builder/bin/gbuild script, it always gets it wrong if there's only one output archive.

Reply:
Yeah, it seems to just be how YAML::Omap in the [gbuild script](https://github.com/devrandom/gitian-builder/blob/3c73ae62b6778fc5776ec4a015b8a401649d55b4/bin/gbuild#L402) handles a single item (wrapping in apostrophes) rather than a list.
For some reason you and I get a few extra spaces and others don't.

Reply:
Ok.... but we're all supposed to be running identical build environments. This implies there's a difference in the environment, which means either depends or gitian is broken somehow. That YAML::Omap is a ruby package, coming from the base OS install. I don't get how this can be different.

Reply:
Oh duh. This is running on the actual host, not in the gitian build container. I'm now on Ubuntu 20.04.1 LTS. The ruby I have here is `ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]` - what's everyone else running?